src/frontend/apps/standalone_site/src/features/Contents/features/Video/components/Update/VideoUpdate.tsx

```diff
@@ -0,0 +1,73 @@
+import { AppConfig, AppConfigProvider, UploadManager } from 'lib-components';
+import { useVideo, DashboardVideoWrapper, useSetVideoState } from 'lib-video';
+import 'lib-video/lib/esm/style.css';
+import { defineMessages, useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+
+import liveBackground from 'assets/img/liveBackground.jpg';
+import { REACT_QUERY_CONF_API } from 'conf/global';
+import { ManageAPIState } from 'features/Contents';
+
+const appConfig = {
+  static: {
+    img: {
+      liveBackground,
+    },
+  },
+} as AppConfig;
+
+const messages = defineMessages({
+  NoVideo: {
+    defaultMessage: 'There is no video to display.',
+    description: 'Text when there is no video to display.',
+    id: 'features.Contents.features.Video.Dashboard.NoVideo',
+  },
+});
+
+const VideoUpdate = () => {
+  const intl = useIntl();
+  const { videoId } = useParams<{ videoId?: string }>();
+
+  if (!videoId) {
+    return (
+      <ManageAPIState
+        isError={false}
+        isLoading={false}
+        hasResult={false}
+        nothingToDisplay={intl.formatMessage(messages.NoVideo)}
+      />
+    );
+  }
+
+  return <VideoDashboard videoId={videoId} />;
+};
+
+const VideoDashboard = ({ videoId }: { videoId: string }) => {
+  const intl = useIntl();
+  const {
+    isError,
+    isLoading,
+    data: currentVideo,
+  } = useVideo(videoId, REACT_QUERY_CONF_API);
+
+  useSetVideoState(currentVideo);
```

Comment:
Should not be done in the `onSuccess` of `useVideo` ?

Reply:
As it is now we cannot, React Hook mustn't be called conditionally.
Inside the hook I control that `currentVideo` is init before setting the states.

Reply:
Indeed, it is not recommended to call hook conditionnaly